metecho/api/serializers.py

```diff
@@ -585,13 +594,15 @@ class TaskSerializer(HashIdModelSerializer):
         required=False,
         allow_null=True,
     )
-    project = serializers.PrimaryKeyRelatedField(
+    project = NestedPrimaryKeyRelatedField(
         queryset=Project.objects.all(),
         pk_field=serializers.CharField(),
+        serializer=ProjectMinimalSerializer,
```

Comment:
Maybe this is misplaced, but I'm hesitant to inline this much data into every serialized task (the list of `github_users` can be very very long for a Project), when most of the time it's duplicated info that we don't need.

From what I can tell, since this new usage of the Tasks table is readonly, we only really care about the `github_user` info for the `assigned_qa` and `assigned_dev` -- could we inline those as nested objects instead of nesting the Project info?